ecosystem.template.json

```diff
@@ -23,6 +23,7 @@
 
                 "BPTF_DETAILS_BUY": "I am buying your %name% for %price%, I have %current_stock% / %max_stock%.",
                 "BPTF_DETAILS_SELL": "I am selling my %name% for %price%, I am selling %amount_trade%.",
+                "OFFER_MESSAGE": " ",
```

Comment:
Make this an empty string, no whitespaces

Reply:
Ok. 👌